src/context.rs

```diff
@@ -1748,6 +1776,345 @@ impl ContextWriter {
     symbol_with_update!(self, w, enable as u32, &mut self.fc.filter_intra_cdfs[block_size as usize]);
   }
 
+  fn get_mvref_ref_frames(&mut self, ref_frame: usize) -> ([usize; 2], usize) {
+    let ref_frame_map: [[usize; 2]; TOTAL_COMP_REFS] = [
+      [ LAST_FRAME,  BWDREF_FRAME  ], [ LAST2_FRAME,  BWDREF_FRAME  ],
```

Comment:
This is fine for now but I'd rather drop all the *_FRAME constants at some point, maybe just with a find/replace here.

Reply:
I agree